### ISSUE TITLE
Password can only be set in "Advanced Configuration", not "Settings".

### ISSUE THREAD

**Issue by terrycloth** [syncthing/syncthing#3413]
I saw a pop-up in my Syncthing web UI, warning that I needed to set a password. This popup includes a button that takes you to the Syncthing "Settings" screen. However, the password cannot be modified there. I could click and type in all the other text fields as normal, but the password field was blank and unresponsive.

I was only able to set a password by going to "Advanced Configuration".

...Also, it's a little odd that, in Advanced Configuration, the username field is a couple lines *below* the password field.

### Version Information

* Syncthing Version: v0.13.10
* OS Version: Fedora 24 (64-bit)
* Browser Version: Firefox 47.0.1

**Comment by calmh** [comment]
I can't reproduce this with my Firefox 43 on Mac (yes I know, old version etc). I would expect more people to complain if it was common.

Anyone else?

**Comment by wweich** [comment]
Can't repro either with Firefox 47 / 49 or Chromium 51 on Windows.

_By the way. The advanced settings are sorted alphanumerically, that's why username and password are apart._

**Comment by terrycloth** [comment]
Weird. I upgraded Syncthing to 0.14 today, and the problem persists.

The GNOME desktop has a feature that lets you write in Japanese by typing the "romaji" -- i.e., Japanese translated to the Latin alphabet -- and then it dynamically replaces the romaji with the appropriate Japanese characters. There's an icon in the top bar of the GNOME Shell that indicates which keyboard setting you're currently using. I just noticed that whenever I try to click in the password field on the Syncthing Settings screen, this GNOME keyboard settings icon flickers. However, I have no problem typing in the password field when my secondary writing language in GNOME is set to German, or to a Japanese keyboard layout that doesn't involve converting romaji to kanji. And I don't have a problem typing in the Syncthing password field, regardless of the GNOME language settings, when using Google Chrome. But I've used the GNOME language settings to let me more easily write Japanese for a long time, and I've never had a problem typing in password fields in Firefox before...

So there's some kind of weird interaction between Firefox, the GNOME romaji-to-kana conversion feature, and the Syncthing Settings password field... The Firefox JavaScript console doesn't reveal any obvious errors when I click on the password field, and I don't see anything unusual about the Syncthing web UI's HTML, so I'm thinking it's probably not a Syncthing bug after all. You guys can close this issue, unless someone here knows more about what's going on than I do.
